com.unity.render-pipelines.universal/Editor/ShaderGraph/UniversalFields.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEditor.ShaderGraph;
+
+namespace UnityEditor.Rendering.Universal.ShaderGraph
+{
+    internal static class UniversalFields
```

Comment:
why are we creating UniversalField? What problem is this solving?

Reply:
Match HDFields.cs in HDRP. It contains Fields that only Universal uses moved from the "core". Shared fields are left in to the "core". 
It's possible that we should align and share more of these, but we should only move any of these back to the "core" when we know it's actually useful and shared.
This was also done to simplify the shared "core" fields, so that we actually know what's shared. The intent is to build the "core" back properly.